docs/using.md

```diff
@@ -171,41 +489,294 @@ is passed in `infrastructure_provisioning: bar` and the third `arbitrary_key: ba
+is typically not known until runtime, this means that most workloads should have a configuration more like the following:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 500
+      Update:
+        QueryOptions:
+          maxPoolSize: 500
+
+
+<a id="orga591018"></a>
+
+### Multiple Connection Strings
+
+If multiple different connection strings are needed, such as when testing a multitenant system, we can use an override file. For the above configuration,
+we can create the following override:
+
+    Clients:
+      Default:
+        URI: "mongodb://localhost:27017"
+      Update:
+        URI: "mongodb://localhost:27018"
+
+Notice the different ports. This can be used at runtime as:
+
+    ./run-genny workload example.yml -o override.yml
+
+This will apply the override onto the workload, creating the following result:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 500
+        URI: "mongodb://localhost:27017"
+      Update:
+        QueryOptions:
+          maxPoolSize: 500
+        URI: "mongodb://localhost:27018"
+
+Genny's `evaluate` subcommand can always be used to see the result of complex configurations.
+
+
+<a id="org65830c2"></a>
+
+### Default
+
+Since actors generally need a connection and not all workloads need a complicated connection or multiple pools,
+simply not setting any connection pools will cause Genny to default to the following:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 100
+        URI: "mongodb://localhost:27017"
+
+For more information, see [Defaults and Overrides](#org22b7a0f).
+
+
+<a id="org7e6c6bd"></a>
+
+## Creating an Actor
+
+Creating new actors is a common and encouraged workflow in Genny. To create one, run the following:
+
+    ./run-genny create-new-actor
+
+This will create new actor .cpp and .h files, an example workload yaml, as well as actor integration tests, all with inline comments guiding you through the actor creation process. You might want to take a look at [Developing Genny](./developing.md) and the [Contribution Guidelines](../CONTRIBUTING.md).
+
+It is encouraged to make new actors as general as possible, for reuse among workloads. No need to loop TIPS into a PR when developing an actor, unless you'd just prefer a second look. Actor authors own their actors.
+
+
+<a id="org3aaae9e"></a>
+
+# Pitfalls
+
+
+<a id="orga7ab911"></a>
+
+## pipe creation failed (24): Too many open files
+
+If you see errors like this locally, try either increasing your ulimit or reducing the number of threads and duration.
+
+
+<a id="orgb084b49"></a>
+
+## Actor integration tests fail locally
+
+There are currently pathing errors when running integration tests locally. This is tracked in [TIG-3687](https://jira.mongodb.org/browse/TIG-3687). That ticket also lists a workaround for local use.
```

Comment:
I..wouldn't call this an error..I would call it a business-logic peculiarity. "There are path differences in how most people check out genny versus how the CI system checks out genny" is probably the most succinct solution. The [evergreen module manager](https://evergreen-ci.github.io/evg-module-manager/) will likely steer us toward how CI checks things out and we can lean into that (assuming we want to lean into emm - topic for another day).